base/doc/ltnews41.tex

```diff
@@ -1078,30 +1282,45 @@ \subsection{\pkg{array}:\ Fix handling of empty p-cells}
 %
 \githubissue{1622}
 
+
 \section{Changes to files in the L3 programming layer}
 
-Work on the L3 programming layer continues in parallel with development of the
-\LaTeX{} kernel. Of note for developers is that we have integrated more code
+%% Should this use subsections? 
+
+Work on the L3 programming layer continues 
+in parallel with development of the
+rest of the
+\LaTeX{} kernel. 
+Of note for developers is that we have integrated more code
 into the main \pkg{l3kernel} bundle, and therefore into the functionality
-available automatically within \LaTeX{}. Most notably, \pkg{l3benchmark}, which
+available automatically in \LaTeX{}. Most notably, \pkg{l3benchmark}, which
 provides tools for checking code performance, is now part of \pkg{l3kernel}.
 
 We have also extended the \pkg{color} module to recognise the Oklab and Oklch
-color model: thanks to Markus Kurtz for contributing this code. The Oklab color
+color models;
+%% Or is there just the one model?
+thanks to Markus Kurtz for contributing this code. The Oklab color
 space (\url{https://bottosson.github.io/posts/oklab}) is a perceptual color
-space which is supported by CSS and so modern web browsers.
+space which is supported by CSS and so also by modern web browsers.
 
 The final \enquote{highlight} here is for developers who want to create
-packages that work not only with \LaTeX{} but also with other formats: generic
-mode \pkg{expl3}. Supporting Con\TeX{}t, in particular when used with
-LuaMeta\TeX{}, requires some work in keeping up with engine changes. After a
-period in which the L3 programming layer was not working there due do
-extensions to (mainly) \cs{numexpr}, work has now been completed to get this
+packages that work not only with \LaTeX{} but also with other formats: 
+generic mode \pkg{expl3}. 
+%% ??? why is this "other formats". should there be an "eg" or "such as" somewhere?
+%% And: what is this "generic mode"?
+Supporting Con\TeX{}t, in particular when used with
+LuaMeta\TeX{}, requires a lot of work in order to keep up with the frequent 
+changes to this engine.  After a
+period in which the L3 programming layer was not working there due do extensions 
+%% extensions where?? and where are these extensions?
```

Comment:
Hans extended the syntax of `\numexpr` to accept `;` as valid input, but that caused an issue as we (Bruno) used it as a non-expandable terminator internally.

Reply:
do extensions -> to LuaMeta\TeX{} engine extensions

otherwise fine imho